### PR TITLE
[webinterface.tex] 1.2.1

### PR DESCRIPTION
**Pull request by clementdelestre**
### Description
TeX is a new web interface for Kodi made with Angular and Tailwind CSS.
Source: https://github.com/clementdelestre/TeX

changelogs :
- Update translations

### Checklist:

- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-webinterfaces/blob/master/CONTRIBUTING.md) document
- [x ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: webinterface.tex/addon.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="webinterface.tex" name="TeX" provider-name="clementdelestre" version="1.2.1">
+	<requires>
+		<import addon="xbmc.json" version="6.0.0" />
+	</requires>
+	<extension point="xbmc.webinterface" />
+	<extension point="xbmc.addon.metadata">
+		<platform>all</platform>
+		<summary lang="en">Modern web interface for Kodi</summary>
```
Comment (enen92) [inline webinterface.tex/addon.xml:9]:
```suggestion
		<summary lang="en">Modern web interface for Kodi</summary>
```
```suggestion
		<summary lang="en_GB">Modern web interface for Kodi</summary>
```

File: webinterface.tex/addon.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		<description lang="en">Browse and interact with your Movies, TV Shows and musics via your web browser.</description>
```
Comment (enen92) [inline webinterface.tex/addon.xml:10]:
```suggestion
		<description lang="en_GB">Browse and interact with your Movies, TV Shows and musics via your web browser.</description>
```

File: webinterface.tex/addon.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+		<platform>all</platform>
+		<language>en</language>
+		<source>https://github.com/clementdelestre/TeX</source>
```
Comment (enen92) [inline webinterface.tex/addon.xml:13]: Can you please add a `<license>` tag with an spdx license identifier?

Comment (clementdelestre) [reply webinterface.tex/addon.xml:13]: Added in new commit.
Is it ok ?

**Comment by enen92** [comment]
yes but all the other requested changes are still missing (the language ones)

**Comment by enen92** [comment]
Also the license.txt specifies GPL-2.0-or-later not AGPL-3.0-or-later

**Pull request closed without merging**